thirdparty/junixsocket/zimbra-junixsocket/rpm/SPECS/junixsocket.spec

```diff
@@ -0,0 +1,44 @@
+Summary:            Zimbra's junixsocket build
+Name:               zimbra-junixsocket
+Version:            2.0.4
+Release:            1zimbra9.0b1
```

Comment:
The version and release lines are incorrect.  Please see other SPEC files for correct format.